.github/workflows/backend-ci.yaml

```diff
@@ -0,0 +1,31 @@
+name: Backend CI
+
+on:
+  pull_request:
+    paths:
+      - "bbk-backend/**"
+    branches: ["*"]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: dotnet build
+    defaults:
+      run:
+        working-directory: bbk-backend
+    steps:
```

Comment:
you could probably just try to build the dockerfile

Reply:
I could but let's leave it as is